### ISSUE TITLE
False positive with implementing several interfaces

### ISSUE THREAD

**Issue by DrAlexD** [saveourtool/diktat#1890]
## Describe the bug
When implementing several interfaces and executing diktat fix, for some reason the interfaces end up on new line.
## Expected behavior
```kotlin
class A<K : Any, P : Any, G : Any> : B<K>(), C<P>, D<G> {}
```
## Observed behavior
```kotlin
class A<K : Any, P : Any, G : Any> : B<K>(),
C<P>,
D<G> {}
```
## Steps to Reproduce
```kotlin
class A<K : Any, P : Any, G : Any> : B<K>(), C<P>, D<G> {}
```

**Comment by diphtongue** [comment]
According to guide (3.6.2.8), if the supertype list has more than two elements, they should be separated by newlines. However, this observed behavior doesn't match the guide: first supertype should also be on the new line.
